trillian/ctfe/instance_test.go

```diff
@@ -388,9 +403,27 @@ func TestValidateLogMultiConfig(t *testing.T) {
 				},
```

Comment:
(nit) The test at line 388-390 is invalid in multiple ways (empty log prefix, backend name that doesn't exist) -- might be nice to only have one error per test case so it's not dependent on the order that things are checked.

Reply:
Done.